src/views/juttle-view.js

```diff
@@ -417,6 +421,20 @@ var JuttleView = Base.extend({
             findFlattenedOptions(optionValidationConfig, '');
 
             return validFlattenedOptions;
+        },
+
+        getFlattenedParamValidationErrors: function(errs) {
```

Comment:
Seems weird that you have to pass in the errors that an instantiated class has. Shouldn't the errors be contained within the class?